RackAndPower/PowerShelfManagementSpec.md

```diff
@@ -0,0 +1,644 @@
+The Power Shelf Management profile is extended from the Baseline Hardware Management profile.
+The following table lists the capabilities provide the baseline profile requirements.
+These capabilities are described in the "Usage Guide and Requirements for the OCP Baseline Hardware Management Profile v1.0.1" document.
+
+| Use Case              | Management Task          | Requirement |
+| :---                  | :---------               | :---        |
+| Account Management    | Get accounts             | Mandatory |
+| Session Management    | Get sessions             | Mandatory |
+| Hardware Inventory    | Get FRU info             | Mandatory |
+|                       | Get and set asset tag    | Recommended |
+| Hardware Location     | Get and set location LED | Recommended |
+| Status                | Get chassis status       | Mandatory |
+| Power                 | Get power state          | Mandatory |
+|                       | Get power usage          | Recommended |
+|                       | Get power limit          | Recommended |
+| Temperature           | Get the temperature      | If implemented, mandatory |
+| Cooling               | Get fan speeds           | If implemented, mandatory |
+|                       | Get fan redundancy       | If implemented, recommended |
+| Log                   | Get log entry            | Mandatory |
+|                       | Clear the log            | Recommended |
+| Management Controller | Get firmware version     | Mandatory |
+|                       | Get controller status    | Mandatory |
+|                       | Get network info         | Mandatory |
+|                       | Reset controller         | Mandatory |
+
+The following table lists the capabilities prescribed in the Power Shelf Management profile.
+
+| Use Case             | Management Task             | Requirement |
+| :---                 | :---------                  | :---        |
+| Power                | Get power supply info       | Mandatory |
+|                      | Get power supply redundancy | If implemented, mandatory |
+|                      | Get power supply metrics    | Mandatory |
+|                      | Get/set power supply LED    | Mandatory |
+|                      | Get power consumption       | Mandatory |
+| Temperature          | Get the temperature         | If implemented, mandatory |
+| Cooling              | Get fan speeds              | If implemented, mandatory |
+|                      | Get fan redundancy          | If implemented, mandatory |
+| Power Equipment      | Get power shelf info        | Mandatory |
+|                      | Get power shelf metrics     | Mandatory |
+|                      | Get main circuits           | Mandatory |
+|                      | Get branch circuits         | Mandatory |
+|                      | Get outlets                 | If implemented, mandatory |
+

```

Comment:
It would be good to add a bubble diagram for those not familiar with the overall Redfish model.  It would allow the reader to relate the odata.id path to something.

Reply:
Done